### PR TITLE
feat(sdks): upgrade to fb-sdk-16

### PR DESCRIPTION
**Pull request by Vincent-liuwingsang**
update FBSDKVersion to 16

### CODE REVIEW

**Comment by kesha-antonov** [comment]
Waiting to get merged, we've received notification - fb wants sdk upgrade

**Review by samRgarcia — APPROVED** [review]

**Comment by kesha-antonov** [comment]
Related issue

https://github.com/facebook/facebook-ios-sdk/issues/2140

File: ios/RCTFBSDK/core/RCTFBSDKAEMReporter.m [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
 #import <React/RCTBridgeModule.h>
 #import <React/RCTUtils.h>
-#import <FBAEMKit/FBAEMReporter.h>
+#import <FBSDKCoreKit/FBSDKCoreKit.h>
```
Comment (kesha-antonov) [inline ios/RCTFBSDK/core/RCTFBSDKAEMReporter.m:10]: I guess it should be ```#import <FBAEMKit/FBAEMKit-Swift.h>```

**Comment by zdnk** [comment]
any update on this?

**Pull request closed without merging**